mmdeploy/codebase/mmdet/models/detectors/base.py

```diff
@@ -6,7 +6,9 @@
 
 
 @mark(
-    'detector_forward', inputs=['input'], outputs=['dets', 'labels', 'masks'])
+    'detector_forward',
+    inputs=['input'],
+    outputs=['dets', 'labels', 'masks', 'semseg'])
```

Comment:
To make the exported model run inference and visualize, we may need to update the `End2EndModel`. Here we suppose three output, may expand to four outputs?

https://github.com/open-mmlab/mmdeploy/blob/fa034e042bb435ad02eedb0961e4672256b42c64/mmdeploy/codebase/mmdet/deploy/object_detection_model.py#L201

Reply:
OK. I'll add an additional operation for treating 4th element of output (semantic segmentation) for panoptic segmentation.